kqueen/blueprints/api/api3_0.yml

```diff
@@ -205,26 +205,180 @@ components:
           type: "string"
         provisioner:
           type: "string"
-          description: "Reference to related Provisioner"
+          description: "Reference to the related Provisioner"
+          example: "Provisioner:<uuid>"
+        state:
+          $ref: "#/components/schemas/clusterState"
+        metadata:
+          description: "GCE-specific parameters for a cluster "
+          type: "object"
+          properties:
+            zone:
+              type: "string"
+              enum:
+                - "us-central1-a"
+                - "us-west1-a"
+                - "us-east1-b"
+                - "us-east4-a"
+                - "northamerica-northeast1-a"
+                - "southamerica-east1-a"
+                - "europe-west1-b"
+                - "europe-west2-a"
+                - "europe-west3-a"
+                - "europe-west4-b"
+                - "asia-northeast1-a"
+                - "asia-east1-a"
+                - "asia-southeast1-a"
+                - "australia-southeast1-a"
+            node_count:
+              type: "integer"
+            machine_type:
+              type: "string"
+              enum:
+                - "n1-standard-1"
+                - "n1-standard-2"
+                - "n1-standard-4"
+                - "n1-standard-8"
+                - "n1-standard-16"
+                - "n1-standard-32"
+                - "n1-standard-64"
+            network_range:
+              type: "string"
+              description: "CIDR block representing network range.
+                            Note, that subnets in the same GCP network must have unique IP ranges."
+              example: "10.0.0.0/14"
+            network_policy:
+              type: "string"
+              description: "To set up network policy you can select Calico network provider"
+              enum:
+                - "PROVIDER_UNSPECIFIED"
+                - "CALICO"
+        created_at:
+          type: "string"
+          format: "date-time"
+        owner:
+          type: "string"
+          description: "Reference to the related User"
+          example: "User:<uuid>"
+      required:
+        - "name"
+        - "provisioner"
+        - "owner"
+        - "metadata"
+    AzureCluster:
+      type: "object"
+      properties:
+        id:
+          type: "string"
+          format: "uuid"
+        name:
+          type: "string"
+        provisioner:
+          type: "string"
+          description: "Reference to the related Provisioner"
           example: "Provisioner:<uuid>"
         state:
           $ref: "#/components/schemas/clusterState"
         kubeconfig:
           description: "Kubeconfig file in JSON format"
           type: "object"
         metadata:
+          description: "Azure-specific parameters, required for a cluster creation"
```

Comment:
+          description: "GCE-specific parameters for a cluster "

imo need to write equal phrases for each entity to avoid possible misunderstandings